### ISSUE TITLE
About [Rewritten "copy [multiple] to selected records"] "improvement"

### ISSUE THREAD

**Issue by Tataarujin** [TES5Edit/TES5Edit#575]
I don't see this as an improvement at all. Instead of doing the same thing the old way and at a lesser time, I have to spend my time to select records, and that time accumulates like a snowball. Please reconsider this feature.

**Comment by ElminsterAU** [comment]
The old way made many mistakes while copying in certain situations and could destroyed data. Sorry, but this is here to stay.

**Comment by Tataarujin** [comment]
> The old way made many mistakes while copying in certain situations and could destroyed data. Sorry, but this is here to stay.

What "certain situations" are you talking about? What percentage of usage these "certain situations" represent?
And, I don't like where this is going, at all. Who's in charge of this "improvement"? Who proposed it? Who approved it? I want to see these people.
Also, seems like this easy explanation of feature won't work because of the change: [https://www.reddit.com/r/xedit/comments/8scu94/select_copypaste_multiple_attributes_at_once/](url)

**Comment by RoyBatty64** [comment]
I wanted this improvement because I often copy single flags to hundreds of records and don't want the other flags broken in the process.

When you work on lots of mods, patches and large projects you want things to easy your development time. So if you want broken behavior, use the old version.

**Comment by Sharlikran** [comment]
Elminster is the original author of xEdit. I picked up the project about 6 years ago with his permission. I am very invested in it. I set up the original SVN and found volunteers. Zilav and I shared in the original decoding of for TES5Edit. I am very attached to this project and I don't feel it too vain for me to say modding would not be what it is without this tool.  

I don't see the need for out team to have anyone that is in charge but if there were it would probably be Elminster.  However, when I see the need, as a means of support for my team members, I exert a certain amount of authority over the project.

Quoting the nexus:

> Within the modding community, you are not inherently entitled to another's time, work, or creations. Comments or posts that imply otherwise are subject to moderation.

The GitHub issues tracker is not a forum. If you have an issue state it plainly and clearly. If you receive an answer that you feel requires clarification then feel free to ask for it. However, comments need to remain civil. I did not appreciate your response and it implied further contemptuous behavior. You are now blocked.
